src/main/java/org/folio/rest/impl/TenantRefAPI.java

```diff
@@ -81,50 +83,85 @@ String servicePointUserFilter(String s) {
     return res;
   }
 
+  /**
+   * TODO: Move to okapi-common for re-use
+   */
+  static class ModuleFrom {
```

Comment:
Migration classes uses `Versioned` for similar logic, see `org.folio.services.migration.BaseMigrationService#versioned`. Should we use `versioned` here as well, or replace with `ModuleFrom` there?

Reply:
Thanks, I've replaced ModuleFrom class by Versioned class.